etc/column/column-docker.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+[DEFAULT]
+# Log file
+log_file = /var/log/column/column.log
+
+# Log info level
+log_level = DEBUG
+
+# API config
+server = 0.0.0.0
```

Comment:
I've brought this up before with the flask listening address.  Listening on 0.0.0.0 is a potential security issue. So defaulting to it is not recommended. Much better to use 127.0.0.1 and let the deployer customize themselves if they are sure 0.0.0.0 is okay.